package.json

```diff
@@ -872,6 +871,7 @@
     "postcss-loader": "^3.0.0",
     "postcss-prefix-selector": "^1.7.2",
     "prettier": "^2.6.1",
+    "pretty-format": "^27.5.1",
```

Comment:
Curious why this change was added?

Reply:
my theory is that it seems there is an issue due to multiple `pretty-format` instances in the project https://github.com/microsoft/TypeScript/issues/29808
so `@testing-library/dom` install the required version inside the `@testing-library/dom/node_modules/pretty-format` and because we import testing library in the "normal" component then the `devDeps` types are not included and that's why we have https://github.com/patrykkopycinski/kibana/blob/chore/renovate-testing-library/x-pack/plugins/observability/public/components/shared/exploratory_view/rtl_helpers.tsx#L11 

Please let me know if that makes sense @jbudz

Reply:
works for me, just wasn't sure why it was needed :)